Sources/Danger/GitHubDSL.swift

```diff
@@ -267,7 +267,7 @@ extension GitHub {
         }
 
         /// Generic UUID.
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 270;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)